### PR TITLE
virttest.qemu_storage: Add snapshot_apply() for qemu_disk_img.snapshot

### PR DESCRIPTION
**Pull request by Chenditang**

### CODE REVIEW

File: virttest/qemu_storage.py [hunk 1]
```diff
@@ -333,6 +333,15 @@ def snapshot_list(self):
 
         return utils.system_output(cmd)
 
+    def snapshot_apply(self):
+        """
+        Apply a snapshot image.
+        """
+        cmd = self.image_cmd
+        cmd += " snapshot -a %s %s" % (self.snapshot_image_filename, self.image_filename)
```
Comment (will-Do) [inline virttest/qemu_storage.py]: I guess you need make sure ```snapshot_image_filename``` exist before apply it

**Comment by Chenditang** [comment]
@will-Do 
Updated, Thanks~~

**Comment by will-Do** [comment]
ACK

**Comment by xutian** [comment]
LGTM

Merged, thanks @Chenditang  and @will-Do

**Pull request merged**